### PR TITLE
fix: add openshift-local release v2.0.0 to extensions catalog

### PR DESCRIPTION
**Pull request by dgolovin**

### CODE REVIEW

File: api/extensions.json [hunk 1]
```diff
@@ -201,6 +201,26 @@
       "categories": ["Kubernetes"],
       "keywords": ["OpenShift", "MicroShift"],
       "versions": [
+        {
+          "version": "1.4.0",
+          "preview": false,
+          "ociUri": "quay.io/redhat-developer/openshift-local-extension:v1.4.0",
```
Comment (dgolovin) [inline api/extensions.json:207]:
```suggestion
          "ociUri": "ghcr.io/crc-org/crc-extension:1.4.0",
```

Comment (dgolovin) [reply api/extensions.json:207]: CC: @jeffmaury

File: api/extensions.json [hunk 2]
```diff
@@ -201,6 +201,26 @@
       "categories": ["Kubernetes"],
       "keywords": ["OpenShift", "MicroShift"],
       "versions": [
+        {
+          "version": "1.4.0",
```
Comment (dgolovin) [inline api/extensions.json:205]:
```suggestion
          "version": "2.0.0",
```

**Review by jeffmaury — CHANGES_REQUESTED** [review]
Folder needs also to be renamed

**Review by dgolovin — COMMENTED** [review]
We got an approval from QA team https://github.com/containers/podman-desktop/issues/8372

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**